src/php/inc/theme-widgets.php

```diff
@@ -6,14 +6,26 @@
  */
 
 /**
- * Register widget area.
+ * Register widgetized areas.
  */
 function sparkpress_theme_widgets_init() {
+	register_sidebar(
+		array(
+			'name'          => 'Primary Widget Area',
+			'id'            => 'primary-widget-area',
+			'description'   => 'Primary sidebar area on the side of the page.',
+			'before_widget' => '<div>',
+			'after_widget'  => '</div>',
+			'before_title'  => '<h3>',
+			'after_title'   => '</h3>',
```

Comment:
From what I could tell, `before_title` and `after_title` don't appear to do anything, so we may be able to exclude them. I think they might have been used with the classic editor, but not with blocks.